files/bash.d.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+function load_bash_fragments() {
+    local frag
+    for frag in ${1%/}/*.sh; do
```

Comment:
Unix allows slack dup.
And Quote to make things simple.

```suggestion
    for frag in "${1}/*.sh"; do
```

In addition, `%%` (longest match) should be applicable:
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html